Source/Details/ASPINRemoteImageDownloader.h

```diff
@@ -47,7 +60,7 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @param preconfiguredPINRemoteImageManager The preconfigured remote image manager that will be used by `sharedDownloader`
  */
-+ (void)setSharedPreconfiguredRemoteImageManager:(nullable PINRemoteImageManager *)preconfiguredPINRemoteImageManager;
++ (void)setSharedPreconfiguredRemoteImageManager:(PINRemoteImageManager *)preconfiguredPINRemoteImageManager;
```

Comment:
If we remove the nullable here it does not match with the implementation. Any reason we removed it?

Reply:
I should change the implementation if I didn't. It makes no sense to call this with `nil`, to support it would be to add a check right away along the lines of `if (preconfiguredPINRemoteImageManager == nil) return;` which is silly.

Reply:
Looks like I did actually also change the implementation so I'm happy with this as is.

Reply:
Please also update the doc comments. The doc still say passing in a nil as the PINRemoteImageManager, it will create a default one.